doc/rtd/development/packaging.rst

```diff
@@ -0,0 +1,105 @@
+.. _building_and_distribution:
+
+Building and Distribution
+*************************
+
+``Cloud-init`` is distributed as a system package and pre-installed in cloud
+images. Correct behavior depends on the tight integration of system services
+performing system-wide configuration during early boot stages. Because of
+this, cloud-init is not published to PyPI as it is not intended to be consumed
+as a pure-python package. Instead, cloud-init uses meson_ to build all
+dependent data files, system-initialization scripts and global configuration
+files.
+
+The build system used by ``cloud-init`` is `meson`_. ``Cloud-init`` is not
+published to pypi.org because it is not intended to be consumed directly as a
+pure-python library, SDK or API for managing cloud configuration.
```

Comment:
The second sentence duplicates content from the previous paragraph.

Reply:
A lot of those paragraphs are duplicated, I just redacted a bunch and restructured the initial paragraph.